[IMP] inventory: gme's package improvements (#7797)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/package.rst

Packages are commonly used for the following purposes:

> #. :ref:`Group products and move them in bulk <inventory/warehouses_storage/pack>`.

Reviewer: ```suggestion
#. :ref:`Grouping products to move them in bulk <inventory/warehouses_storage/pack>`.
```

*Package use* is a field on the package form in Odoo that is only visible by enabling the *Batch

> Transfers* feature in settings of the *Inventory* application.

Reviewer: ```suggestion
Transfers* feature in the settings of the *Inventory* application.
```

> Transfers* feature in settings of the *Inventory* application.

Reviewer 2: Should we include a quick instruction on how to enable that, if the reader doesn't have it enabled already?

   :align: center
   :alt: Activate the *Packages* setting in Inventory > Configuration > Settings.

.. _inventory/warehouses_storage/pack:

Pack items
==========

Products can be added to packages in any transfer by:

> #. clicking each *Detailed Operations* icon on the product line
> #. using the *Put in Pack* button to place everything in the transfer into a package.

Reviewer: ```suggestion
#. Clicking each *Detailed Operations* icon on the product line.
#. Using the *Put in Pack* button to place everything in the transfer into a package.
```

Detailed operations
-------------------

On any transfer, add a product to a package by clicking the :guilabel:`⦙≣ (bulleted list)` icon in
the :guilabel:`Operations` tab.

.. image:: package/detailed-operations.png
   :align: center
   :alt: Show "Detailed Operations" icon in the product line.

Doing so opens the :guilabel:`Detailed Operations` pop-up window for the :guilabel:`Product`.

To put the :guilabel:`Product` in a package, click :guilabel:`Add a line`, and assign the product to

> a :guilabel:`Destination Package`. Select an existing package or create a new one by typing the name

Reviewer 2: ```suggestion
a :guilabel:`Destination Package`. Select an existing package, or create a new one by typing the name
```

> of the package and select the :guilabel:`Create...`.

Reviewer: ```suggestion
of the package and select :guilabel:`Create...`.
```

> of the package and select the :guilabel:`Create...`.

Reviewer 2: ```suggestion
of the package and selecting the :guilabel:`Create...` button.
```

Reviewer 2: or: "...of the package and selecting :guilabel:`Create`.

.. figure:: package/destination-package.png
   :align: center
   :alt: Assign a package to "Destination Package" field.

   Twelve units of `Acoustic Bloc Screen` are placed in `PACK0000001`.

Then, specify the quantity of items to go into the package in the :guilabel:`Done` field. Repeat the
above steps to place the :guilabel:`Product` in different packages. Once finished, click
:guilabel:`Confirm` to close the window.

.. seealso::
   :doc:`Ship one order in multiple packages
   <../../shipping_receiving/advanced_operations_shipping/multipack>`

Put in pack
-----------

Alternatively, click the :guilabel:`Put in Pack` button on any transfer to create a new package, and
place all the items in the transfer in that newly-created package.

.. figure:: package/put-in-pack.png
   :align: center
   :alt: Image of the "Put in Pack" button being clicked.

>    In a batch transfer `BATCH/00003` the :guilabel:`Put in Pack` button was clicked to create a new

Reviewer: ```suggestion
   In batch transfer `BATCH/00003`, the :guilabel:`Put in Pack` button was clicked to create a new
```

   package, `PACK0000002`, and assign all items to it in the :guilabel:`Destination Package` field.

.. _inventory/warehouses_storage/package-type:

Package type
============

Create package types at :menuselection:`Inventory app --> Configuration --> Package Types` to set
custom dimensions and weight limits. This feature is mainly used to calculate package weights for
shipping costs.

.. seealso::
   - :doc:`Shipping carriers <../../shipping_receiving/setup_configuration/third_party_shipper>`
   - :doc:`../../shipping_receiving/setup_configuration/delivery_method`

On the :guilabel:`Package Types` list, clicking :guilabel:`New` opens a blank package type form. The
fields of the form are as follows:

- :guilabel:`Package Type` (required): define the package type's name.
- :guilabel:`Size`: define the dimensions of the package in millimeters (mm). The fields, from left
  to right, define the :guilabel:`Length`, :guilabel:`Width`, and :guilabel:`Height`.

> - :guilabel:`Weight`: Weight of an empty package (e.g. an empty box, pallet).

Reviewer 2: ```suggestion
- :guilabel:`Weight`: weight of an empty package (e.g. an empty box, pallet).
```

.. note::
   Odoo calculates the package's weight by adding the weight of the empty package with the weight of

>    the item (from the :guilabel:`Weight` field in the :guilabel:`Inventory` tab of the product form)

Reviewer: ```suggestion
   the item(s) (from the :guilabel:`Weight` field in the :guilabel:`Inventory` tab of each product form).
```

>    Odoo calculates the package's weight by adding the weight of the empty package with the weight of
>    the item (from the :guilabel:`Weight` field in the :guilabel:`Inventory` tab of the product form)

Reviewer 2: ```suggestion
   Odoo calculates the package's weight by adding the weight of the empty package plus the weight of
   the item (from the :guilabel:`Weight` field in the :guilabel:`Inventory` tab of the product form)
```

.. image:: package/package-type.png
   :align: center
   :alt: Package type for FedEx's 25 kilogram box.

.. _inventory/warehouses_storage/cluster-pack:

Cluster packages
================

> First, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and activate the

Reviewer 2: ```suggestion
To use *cluster packages*, first navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and activate the
```
